src/markable/converter.c

```diff
@@ -65,6 +65,59 @@ static Janet cfun_markdown_to_html(int32_t argc, Janet *argv) {
     return janet_wrap_string(html);
 }
 
+/* Wrap cmark-gfm's cmark_render_plaintext_with_mem().
+ *
+ * This function expects argv to contain between one and three values.
+ *
+ * The first value is the Markdown-formatted string to be converted to
+ * plaintext.
+ *
+ * The optional second value is a JanetTuple collection of JanetKeyword values
+ * that represent the options to used to parse the string. If the second
+ * parameter is not provided, the current value of markable_default_options
+ * will be used.
+ *
+ * The optional third value is an unsigned integer representing the width at
+ * which to wrap the resulting output.
+ *
+ * The return value is a wrapped JanetString value.
+ */
+static Janet cfun_markdown_to_plaintext(int32_t argc, Janet *argv) {
+    janet_arity(argc, 1, 3);
+
+    const char *input = janet_getcstring(argv, 0);
+
+    int32_t options;
+    if (argc >= 2) {
+        options = markable_extract_options(janet_getindexed(argv, 1));
+    } else {
+        options = markable_default_options;
+    }
+
+    uint32_t width;
+    if (argc == 3) {
+        width = janet_getuinteger(argv, 2);
+    } else {
+        width = 80;
+    }
+
+    cmark_mem *arena_mem = cmark_get_arena_mem_allocator();
+    cmark_parser *parser = cmark_parser_new_with_mem(options, arena_mem);
+
+    cmark_parser_feed(parser, input, strlen(input));
+    cmark_node *doc = cmark_parser_finish(parser);
+
+    cmark_mem *default_mem = cmark_get_default_mem_allocator();
+    char *output = cmark_render_plaintext_with_mem(doc, options, width, default_mem);
```

Comment:
@sogaiu This must be something to do with the default compilation settings in Clang but when I tried to run the code on macOS, I got an error because `cmark_render_plaintext_with_mem` doesn't take a pointer to an extensions struct but instead takes a width (I think at which to wrap the output). My changes are generally focused on fixing that.

Reply:
My bad, I didn't look closely enough (^^;

Thanks!